index.html

```diff
@@ -390,7 +463,7 @@ <h2 class="contact-title">Contact me</h2>
                         placeholder="Write your message here..." type="text"></textarea>
                 </div>
 
-                <div class="">
+                <div class="contact-form-button-container">
```

Comment:
Please use the right background color for the `hover` interaction of the `Get in touch` button of the `contact me` section to help you with a better UI : 

Button interaction :

![image](https://user-images.githubusercontent.com/20537712/236193226-1424b242-3001-4315-8632-88568688f8b7.png)